cli/Makefile

```diff
@@ -2,20 +2,23 @@ build: tools embed_files lint build-amd64
 
 build-all: tools embed_files lint build-amd64 build-arm64 build-arm32 build-windows build-darwin
 
+build-all-small:
+	@$(MAKE) LDFLAGS+="-s -w" build-all
+
 build-arm32: lint
-	GOARCH="arm" GOOS="linux" go build -o dist/carrier-linux-arm32
+	GOARCH="arm" GOOS="linux" go build -ldflags '$(LDFLAGS)' -o dist/carrier-linux-arm32
 
 build-arm64: lint
-	GOARCH="arm64" GOOS="linux" go build -o dist/carrier-linux-arm64
+	GOARCH="arm64" GOOS="linux" go build -ldflags '$(LDFLAGS)' -o dist/carrier-linux-arm64
 
 build-amd64: lint
-	GOARCH="amd64" GOOS="linux" go build -race -o dist/carrier-linux-amd64
```

Comment:
Was `-race` removed on purpose because it was added explicitly: 6467cd701cb99421f9c6f66356386496b8bdecca

Reply:
Yes, I removed by mistake. Corrected it.